<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Calculator</class>
 <widget class="QMainWindow" name="Calculator">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>330</width>
    <height>500</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>330</width>
    <height>500</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>330</width>
    <height>500</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Calculator</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="MathBtnMultiply">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>180</y>
      <width>61</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color: rgb(219, 216, 255);
	border:  none;
	border-radius: 5px;
	color: black;
}

QPushButton:pressed {
	background-color: rgb(161, 170, 255);
	border:  none;
	border-radius: 5px;
	color: black;
}

QPushButton:hover {
	background-color: rgb(161, 170, 255);
	border:  none;
	border-radius: 5px;
	color: black;
}</string>
    </property>
    <property name="text">
     <string>*</string>
    </property>
   </widget>
   <widget class="QPushButton" name="MathBtnDivide">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>120</y>
      <width>61</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color: rgb(219, 216, 255);
	border:  none;
	border-radius: 5px;
	color: black;
}

QPushButton:pressed {
	background-color: rgb(161, 170, 255);
	border:  none;
	border-radius: 5px;
	color: black;
}

QPushButton:hover {
	background-color: rgb(161, 170, 255);
	border:  none;
	border-radius: 5px;
	color: black;
}</string>
    </property>
    <property name="text">
     <string>/</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Button8">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>180</y>
      <width>61</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color: rgb(219, 216, 255);
	border:  none;
	border-radius: 5px;
	color: black;
}

QPushButton:pressed {
	background-color: rgb(161, 170, 255);
	border:  none;
	border-radius: 5px;
	color: black;
}

QPushButton:hover {
	background-color: rgb(161, 170, 255);
	border:  none;
	border-radius: 5px;
	color: black;
}</string>
    </property>
    <property name="text">
     <string>8</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Button2">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>300</y>
      <width>61</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color: rgb(219, 216, 255);
	border:  none;
	border-radius: 5px;
	color: black;
}

QPushButton:pressed {
	background-color: rgb(161, 170, 255);
	border:  none;
	border-radius: 5px;
	color: black;
}

QPushButton:hover {
	background-color: rgb(161, 170, 255);
	border:  none;
	border-radius: 5px;
	color: black;
}</string>
    </property>
    <property name="text">
     <string>2</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Btn1bySth">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>120</y>
      <width>61</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color: rgb(219, 216, 255);
	border:  none;
	border-radius: 5px;
	color: black;
}

QPushButton:pressed {
	background-color: rgb(161, 170, 255);
	border:  none;
	border-radius: 5px;
	color: black;
}

QPushButton:hover {
	background-color: rgb(161, 170, 255);
	border:  none;
	border-radius: 5px;
	color: black;
}</string>
    </property>
    <property name="text">
     <string>1/x</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Button1">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>300</y>
      <width>61</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color: rgb(219, 216, 255);
	border:  none;
	border-radius: 5px;
	color: black;
}

QPushButton:pressed {
	background-color: rgb(161, 170, 255);
	border:  none;
	border-radius: 5px;
	color: black;
}

QPushButton:hover {
	background-color: rgb(161, 170, 255);
	border:  none;
	border-radius: 5px;
	color: black;
}</string>
    </property>
    <property name="text">
     <string>1</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="Display">
    <property name="geometry">
     <rect>
      <x>32</x>
      <y>47</y>
      <width>261</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color: rgb(219, 216, 255);
	border:  none;
	border-radius: 5px;
	color: black;
}

QPushButton:pressed {
	background-color: rgb(161, 170, 255);
	border:  none;
	border-radius: 5px;
	color: black;
}

QLineEdit{
	background-color: rgb(226, 226, 226);
	border:  1px solid #000;
	border-radius: 5px;
	color: black;
	padding: 10px;
}</string>
    </property>
    <property name="text">
     <string>0.0</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="BtnPlusMinus">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>360</y>
      <width>61</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color: rgb(219, 216, 255);
	border:  none;
	border-radius: 5px;
	color: black;
}

QPushButton:pressed {
	background-color: rgb(161, 170, 255);
	border:  none;
	border-radius: 5px;
	color: black;
}

QPushButton:hover {
	background-color: rgb(161, 170, 255);
	border:  none;
	border-radius: 5px;
	color: black;
}</string>
    </property>
    <property name="text">
     <string>+/-</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ButtonPoint">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>360</y>
      <width>61</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color: rgb(219, 216, 255);
	border:  none;
	border-radius: 5px;
	color: black;
}

QPushButton:pressed {
	background-color: rgb(161, 170, 255);
	border:  none;
	border-radius: 5px;
	color: black;
}

QPushButton:hover {
	background-color: rgb(161, 170, 255);
	border:  none;
	border-radius: 5px;
	color: black;
}</string>
    </property>
    <property name="text">
     <string>.</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Button6">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>240</y>
      <width>61</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color: rgb(219, 216, 255);
	border:  none;
	border-radius: 5px;
	color: black;
}

QPushButton:pressed {
	background-color: rgb(161, 170, 255);
	border:  none;
	border-radius: 5px;
	color: black;
}

QPushButton:hover {
	background-color: rgb(161, 170, 255);
	border:  none;
	border-radius: 5px;
	color: black;
}</string>
    </property>
    <property name="text">
     <string>6</string>
    </property>
   </widget>
   <widget class="QPushButton" name="BtnPow2">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>120</y>
      <width>61</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color: rgb(219, 216, 255);
	border:  none;
	border-radius: 5px;
	color: black;
}

QPushButton:pressed {
	background-color: rgb(161, 170, 255);
	border:  none;
	border-radius: 5px;
	color: black;
}

QPushButton:hover {
	background-color: rgb(161, 170, 255);
	border:  none;
	border-radius: 5px;
	color: black;
}</string>
    </property>
    <property name="text">
     <string>x^2</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Button7">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>180</y>
      <width>61</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color: rgb(219, 216, 255);
	border:  none;
	border-radius: 5px;
	color: black;
}

QPushButton:pressed {
	background-color: rgb(161, 170, 255);
	border:  none;
	border-radius: 5px;
	color: black;
}

QPushButton:hover {
	background-color: rgb(161, 170, 255);
	border:  none;
	border-radius: 5px;
	color: black;
}</string>
    </property>
    <property name="text">
     <string>7</string>
    </property>
   </widget>
   <widget class="QPushButton" name="MathBtnSubtract">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>240</y>
      <width>61</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color: rgb(219, 216, 255);
	border:  none;
	border-radius: 5px;
	color: black;
}

QPushButton:pressed {
	background-color: rgb(161, 170, 255);
	border:  none;
	border-radius: 5px;
	color: black;
}

QPushButton:hover {
	background-color: rgb(161, 170, 255);
	border:  none;
	border-radius: 5px;
	color: black;
}</string>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Button3">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>300</y>
      <width>61</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color: rgb(219, 216, 255);
	border:  none;
	border-radius: 5px;
	color: black;
}

QPushButton:pressed {
	background-color: rgb(161, 170, 255);
	border:  none;
	border-radius: 5px;
	color: black;
}

QPushButton:hover {
	background-color: rgb(161, 170, 255);
	border:  none;
	border-radius: 5px;
	color: black;
}</string>
    </property>
    <property name="text">
     <string>3</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Button4">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>240</y>
      <width>61</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color: rgb(219, 216, 255);
	border:  none;
	border-radius: 5px;
	color: black;
}

QPushButton:pressed {
	background-color: rgb(161, 170, 255);
	border:  none;
	border-radius: 5px;
	color: black;
}

QPushButton:hover {
	background-color: rgb(161, 170, 255);
	border:  none;
	border-radius: 5px;
	color: black;
}</string>
    </property>
    <property name="text">
     <string>4</string>
    </property>
   </widget>
   <widget class="QPushButton" name="MathBtnPlus">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>300</y>
      <width>61</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color: rgb(219, 216, 255);
	border:  none;
	border-radius: 5px;
	color: black;
}

QPushButton:pressed {
	background-color: rgb(161, 170, 255);
	border:  none;
	border-radius: 5px;
	color: black;
}

QPushButton:hover {
	background-color: rgb(161, 170, 255);
	border:  none;
	border-radius: 5px;
	color: black;
}</string>
    </property>
    <property name="text">
     <string>+</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Button9">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>180</y>
      <width>61</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color: rgb(219, 216, 255);
	border:  none;
	border-radius: 5px;
	color: black;
}

QPushButton:pressed {
	background-color: rgb(161, 170, 255);
	border:  none;
	border-radius: 5px;
	color: black;
}

QPushButton:hover {
	background-color: rgb(161, 170, 255);
	border:  none;
	border-radius: 5px;
	color: black;
}</string>
    </property>
    <property name="text">
     <string>9</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ButtonEquals">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>360</y>
      <width>61</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color: rgb(194, 255, 187);
	border:  none;
	border-radius: 5px;
	color: black;
}</string>
    </property>
    <property name="text">
     <string>=</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ButtonClear">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>120</y>
      <width>61</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color: rgb(219, 216, 255);
	border:  none;
	border-radius: 5px;
	color: black;
}

QPushButton:pressed {
	background-color: rgb(161, 170, 255);
	border:  none;
	border-radius: 5px;
	color: black;
}

QPushButton:hover {
	background-color: rgb(161, 170, 255);
	border:  none;
	border-radius: 5px;
	color: black;
}</string>
    </property>
    <property name="text">
     <string>С</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Button5">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>240</y>
      <width>61</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color: rgb(219, 216, 255);
	border:  none;
	border-radius: 5px;
	color: black;
}

QPushButton:pressed {
	background-color: rgb(161, 170, 255);
	border:  none;
	border-radius: 5px;
	color: black;
}

QPushButton:hover {
	background-color: rgb(161, 170, 255);
	border:  none;
	border-radius: 5px;
	color: black;
}</string>
    </property>
    <property name="text">
     <string>5</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Button0">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>360</y>
      <width>61</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color: rgb(219, 216, 255);
	border:  none;
	border-radius: 5px;
	color: black;
}

QPushButton:pressed {
	background-color: rgb(161, 170, 255);
	border:  none;
	border-radius: 5px;
	color: black;
}

QPushButton:hover {
	background-color: rgb(161, 170, 255);
	border:  none;
	border-radius: 5px;
	color: black;
}</string>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>330</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
